### ISSUE TITLE
Not working on HavocOS - OnePlus 6T (fajita)

### ISSUE THREAD

**Issue by Adiker** [programminghoch10/MobileIconChanger#5]
The icon just doesn't change. I'm using LSPosed 1.5.3 with resource hooks enabled. The ROM has network indicators but no way to turn them off. Thanks for any help provided.
![Screenshot_20210919-150429_MobileIconChanger](https://user-images.githubusercontent.com/40745238/133928628-1c207893-d5fa-4c1d-98a8-fe35a4c79b38.png)

**Comment by programminghoch10** [comment]
Yeah as said in the readme, there is nothing I can do about it. Obviously HavocOS also introduces a lot of changes which are not compatible with this mod. I'm sorry.
